client/ayon_silhouette/api/workio.py

```diff
@@ -0,0 +1,38 @@
+"""Host API required Work Files tool"""
+import fx
+
+
+def _get_project() -> "fx.Project":
+    return fx.activeProject()
+
+
+def file_extensions():
```

Comment:
This is used only on host class, no reason to have it here.

Reply:
It's mostly to align it with the layout of other host integrations that are out there. The less we deviate the more it appears similar, but maybe we should update indeed and streamline here regardless of what we're doing in other hosts.

Will remove.

Reply:
The other hosts didn't move it to the class because they don't want to deviate, so it will never happen. We've added `IWorkfileHost` to get rid of `workio`, but failed miserably.

Reply:
https://github.com/ynput/ayon-silhouette/pull/1/commits/b62cb2d4c835260f06df6733fd67002dc8b9b9e7